### PR TITLE
add photo and style in section2

### PR DESCRIPTION
**Pull request by amfever505**

### CODE REVIEW

File: components/ImageLightbox/styles.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
+img {
+  height: 100%;
+}
```
Comment (oasis10702) [inline components/ImageLightbox/styles.scss]: suggest adding blank between selector and selector

File: components/ImageLightbox/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        style={{
+          position: 'absolute',
+          top: '50%',
+          left: '50%',
+          transform: 'translate(-50%, -50%)',
+          color: '#fff',
+          zIndex: 2
+        }}
+      >
+        <h3 className="city_text">Tokyo</h3>
```
Comment (oasis10702) [inline components/ImageLightbox/index.js]: h3 is needless

**Comment by oasis10702** [comment]
ok. rebase

**Comment by amfever505** [comment]
I had rewritten scss constrcture in section2 and updated image file from horizontalsection to assets.

**Pull request merged**